taxi/forms.py

```diff
@@ -0,0 +1,53 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Car
+
+
+class MixinLicenseValidationForm(forms.ModelForm):
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+
+        if len(license_number) != 8:
+            raise ValidationError("Should be consist only of 8 characters")
+
+        if (not license_number[:3].isalpha()
+                or not license_number[:3].isupper()):
+            raise ValidationError("First 3 characters are uppercase letters")
+
+        if not license_number[-5:].isdigit():
+            raise ValidationError("Last 5 characters are digits")
+
+        return license_number
```

Comment:
inheritance is redundant here, put mixin in the end of the name

```suggestion
class LicenseValidationMixin:
    def clean_license_number(self):
        license_number = self.cleaned_data["license_number"]

        if len(license_number) != 8:
            raise ValidationError("Should be consist only of 8 characters")

        if (not license_number[:3].isalpha()
                or not license_number[:3].isupper()):
            raise ValidationError("First 3 characters are uppercase letters")

        if not license_number[-5:].isdigit():
            raise ValidationError("Last 5 characters are digits")

        return license_number
```